quickfort.lua

```diff
@@ -151,6 +151,26 @@ For more info, see: https://docs.dfhack.org/en/stable/docs/_auto/base.html#quick
 ]]
 end
 
+local function set_setting(key, value)
+    if quickfort_common.settings[key] == nil then
+        error(string.format('error: invalid setting: "%s"', key))
```

Comment:
You could probably use [qerror](https://docs.dfhack.org/en/stable/docs/Lua%20API.html#exception-handling) for user-facing error messages like these

Reply:
thanks! done.